Polli/Reinicio.py

```diff
@@ -0,0 +1,115 @@
+from dataset_alunas import dataset
+
+dataset
+
+def main():
+    print("\n---  Seja bem vinda a Escola do Reprograma!  ---")
+    print("Sistema de informações de alunas")
+
+    while True:
+        cod_opcao = obter_opcao()
+        
+        if cod_opcao == 1:
+            incluir_nova_aluna()
+        elif cod_opcao == 2:
+            consultar_lista_alunas()
+        elif cod_opcao == 3:
+            consultar_faltas_aluna()
+        elif cod_opcao == 4:
+            consultar_notas_aluna()
+        elif cod_opcao == 5:
+            consultar_status_aprovacao()
+        elif cod_opcao == 6:
+            print("Encerrando o programa...")
+            break
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+    return codigo_opcao
+
+    
+def incluir_nova_aluna():
+    print("Insira os seguintes dados: ")
+    nome = str(input("Nome da aluna: ")) #Recebo nome da aluna
+    sobrenome = str(input("Sobrenome da aluna: ")) #Recebo sobrenome da aluna
+    turma = int(input("Turma da aluna (nº): ")) #Recebo nome a turma
+    lista_notas = (input('Digite as notas da aluna separadas por vírgula: '))
```

Comment:
Por mais que esteja separando por vírgula, o valor de entrada é uma string. Assim, você terá um valor parecido com esse na sua variável lista_notas:

lista_notas = "10, 5, 2, 5"

Conforme vimos na aula de sábado, é esperado que as notas estejam dentro de uma lista. Revise a aula para entender como funcionam a manipulação de listas, Aqui a ideia é treinar a manipulação e utilização de diversas estruturas de dados.

**Sugestão:**
```suggestion
    lista_notas = obter_notas()
    
def obter_notas():
    quantidade_notas = int(input('Quantas notas serão calculadas?: '))
    notas = []
    for contador in range(quantidade_notas):
        while True: 
            entrada = input(f'Nota do #{contador + 1}: ')
            try:
                nota = float(entrada)
                notas.append(nota)
                break
            except ValueError:
                print('Erro em processar o valor. Insira um número válido')

    return notas
    
    
```